src/main/resources/application.yml

```diff
@@ -24,8 +24,15 @@ spring:
           consumer:
             auto-bind-dlq: true
             max-attempts: 1
+        bindings:
```

Comment:
you also need to define spring.cloud.stream.bindings.consumeTaskDtoUpdate-in-0.destination to precise the exchange name